cmd/res/devices/devices.toml.example

```diff
@@ -1,51 +1,7 @@
-[Writable]
```

Comment:
I agree that this is inconsistent. Prior to this PR we had two configuration.toml files, one in /res, and one in /res/examples. This PR takes the latter file, strips everything but the `DeviceList` (which the top-level TOML file lacks) and re-names `devices.toml.example`. The former device profile however is moved into a new directory `/res/profile`, but is left as `.yml` file, so it will be imported on startup. I think we should be consistent and either have both use a `.example` suffix to prevent them from being auto-imported, or don't add the suffix and have both files processed by default (which means the service will inherit the devices as defined in the `devices.toml` file.

Reply:
you're right it's inconsistent in this PR (profiles are auto-imported while devices are not).

I think we can remove the .example suffix and update the `ProfilesDir` and `DevicesDir` to just `./res`. If user want to import the example devices/profiles, they can env overwrite those 2 configuration.

@lenny-intel @tonyespy thought on this solution ?

Reply:
Seems ok to me. I'd be tempted to rename devices.toml to modbus.test.devices.toml for clarity and consistency with the profile yml